src/pages/usage/custom-resource-apis.md

```diff
@@ -6,7 +6,7 @@ slug: status
 toc: true
 ---
 
-The Annotator for Clinical Data Container Edition is a Custom Resource Definition used to deploy and manage an instance of IBM Watson Annotator for Clinical Data Service.
+The `Acd` resource is a Custom Resource Definition used to deploy and manage an instance of IBM Watson Annotator for Clinical Data Service. When a custom resource is created or updated, the declared state (or Spec field) is updated and available immediately. Over time, the system will work to bring the Status into line with the Spec. These fields provide a standard way to report the state of the custom resource or ACD Service instance.
```

Comment:
Should Acd be ACD here?

Reply:
The resource or kind of object is actually "Acd" to align with naming conventions.
```
kind: CustomResourceDefinition
metadata:
  creationTimestamp: null
  labels:
    app.kubernetes.io/instance: ibm-wh-acd-operator
    app.kubernetes.io/managed-by: ibm-wh-acd-operator
    app.kubernetes.io/name: ibm-wh-acd-operator
  name: acds.wh-acd.ibm.com
spec:
  group: wh-acd.ibm.com
  names:
    kind: Acd
    listKind: AcdList
    plural: acds
    singular: acd
```